src/bijectors/stacked.jl

```diff
@@ -47,7 +47,9 @@ isclosedform(b::Stacked) = all(isclosedform, b.bs)
 
 isinvertible(b::Stacked) = all(isinvertible, b.bs)
 
-stack(bs...) = Stacked(bs)
+
+@deprecate stack stack_transforms
```

Comment:
Doesn't this mean users still have a name-clash when loading Bijectors?

Also generally, I think we should keep the existing tests when deprecating to make sure that the deprecated function still works.

Reply:
> Doesn't this mean users still have a name-clash when loading Bijectors?

Oh crap, I forgot that `@deprecate` exports by default. Should we not export? (that was my original intention)

> Also generally, I think we should keep the existing tests when deprecating to make sure that the deprecated function still works.

Good point; will do!